src/syscheckd/whodata/win_whodata.c

```diff
@@ -150,27 +150,28 @@ int set_winsacl(const char *dir, int position) {
     ZeroMemory(&old_sacl_info, sizeof(ACL_SIZE_INFORMATION));
 
     // Check if the sacl has what the whodata scanner needs
-    switch(is_valid_sacl(old_sacl, (syscheck.wdata.dirs_status[position].object_type == WD_STATUS_FILE_TYPE) ? 1 : 0)) {
-        case 0:
-            mdebug1(FIM_SACL_CHECK_CONFIGURE, dir);
-            syscheck.wdata.dirs_status[position].status |= WD_IGNORE_REST;
+    switch (is_valid_sacl(old_sacl, (syscheck.wdata.dirs_status[position].object_type == WD_STATUS_FILE_TYPE) ? 1 : 0)) {
```

Comment:
I know this was not changed in this PR but is really hard to understand, could we refactor it to something like this? The definition of `is_file` can be moved up in the function.
```suggestion
    int is_file = syscheck.wdata.dirs_status[position].object_type == WD_STATUS_FILE_TYPE) ? 1 : 0;
    
    switch (is_valid_sacl(old_sacl, is_file) {
```